### ISSUE TITLE
HelpDot image request for 5 Expensify Classic articles

### ISSUE THREAD

**Issue by ren-jones** [Expensify/App#49458]
Please provide the following images in light mode and in .png format.

**TAX TRACKING**

Link to resource: https://help.expensify.com/articles/expensify-classic/workspaces/Tax-Tracking

2 Images needed: 

- For a workspace connected to Xero, QuickBooks Online, Sage Intacct, or NetSuite, go to Settings > Workspaces > Group > [Workspace Name] > Connections > Configure, then add the orange highlight beside the enable tax toggle/option. 
- For a workspace not connected to an accounting integration, go to Settings > Workspaces > Group > [Workspace Name] > Tax, then add the orange highlight beside the enable tax toggle.

**THE EXPENSES PAGE**

Link to resource: https://help.expensify.com/articles/expensify-classic/expenses/The-Expenses-Page

5 Images needed:

- Show the Expenses tab with the orange highlight next to the Expenses tab. (This one is just to show the Expenses page and how you get to it. Please make sure realistic expenses are present on the Expenses page)
- Show the Expenses tab with a box or other highlight around the whole top section of the screen where the filters are (to show that all of those items are filters)
- On the Expenses tab, select a couple expenses and add the orange highlight next to the Add to Report button. 
- On the Expenses tab, click an empty Category or Tag cell for an expense, then add the orange highlight next to the search box with the dropdown menu showing (to highlight that you can just click these cells to add a category/tag/description)
- On the Expenses tab, select a couple expenses and add the orange highlight next to the Export To button. 

**CSV IMPORT**

Link to resource: https://help.expensify.com/articles/expensify-classic/connect-credit-cards/company-cards/CSV-Import

5 Images needed: 

- Show the CSV [template](https://s3-us-west-1.amazonaws.com/concierge-responses-expensify-com/uploads%2F1594908368712-Best+Example+CSV+for+Domains.csv) with a few rows filled out as an example. The file should include the following columns and data: 
   - Card Number
   - Posted Date (YYYY-MM-DD format)
   - Merchant
   - Posted Amount (number format, indicate negative amounts with parentheses)
   - Posted Currency (use currency codes, e.g., USD, GBP, EUR)
- Go to Settings > Domains > [Domain Name] > Company Cards and add the orange highlight next to the Manage/Import CSV button.
- Go to Settings > Domains > [Domain Name] > Company Cards, click the dropdown under the Imported Cards title and add the orange highlight next to whatever is showing that you clicked to expand the dropdown (we want to show that you can click this and make a dropdown appear where you can select an option).
- Follow the same steps in Step 3 above but then select one of the CSV import files from the dropdown. Then add the orange highlight next to the Assign New Cards button. 
- Go to Settings > Domains > [Domain Name] > Company Cards, click Manage/Import CSV, select the saved layout from the drop-down list, click Upload CSV, then add the orange highlight next to the Update All Cards button.

**COMMERCIAL CARD FEEDS**

Link to resource: https://help.expensify.com/articles/expensify-classic/connect-credit-cards/company-cards/Commercial-Card-Feeds

2 Image needed: 

- For the second image that is currently in the article, add the orange highlight next to the Employee Email dropdown.
- Go to Settings > Domains > [Domain Name] > Company Cards, then add the orange highlight next to the Edit Exports button for one of the items in the table.

**COMPANY CARD SETTINGS**

Link to resource: https://help.expensify.com/articles/expensify-classic/connect-credit-cards/company-cards/Company-Card-Settings

3 Images needed:

- Go to Settings > Domains > [Domain Name] > Company Cards, then add the orange highlight next to the Settings tab.
- Go to Settings > Workspaces > Group > [Workspace Name] > Reports, then add the orange highlight next to the Scheduled Submit toggle.
- For a workspace that has a pooled GL account, go to Settings > Workspaces > Group > [Workspace Name] > Connections > Expand the accounting integration, then add the orange highlight next to the Credit Card / Charge Card / Bank Transaction selector for the Non-reimbursable export option.

**Comment by dubielzyk-expensify** [comment]
Under CSV import, can you clarify this? 
>Follow the same steps in Step 3 above but then select one of the CSV import files from the dropdown. Then add the orange highlight next to the Assign New Cards button.
Go to Settings > Domains > [Domain Name] > Company Cards, click Manage/Import CSV, select the saved layout from the drop-down list, click Upload CSV, then add the orange highlight next to the Update All Cards button.

I don't see a `Assign New Cards` button and the "follow the same steps in Step 3 above" is a bit unclear. Would be helpful if I could get the full instruction.  I also don't see `Update All Cards` button.

**Comment by ren-jones** [comment]
@dubielzyk-expensify Ah, these steps were based on an older article, so those areas may have changed since then (I'm unsure because I wasn't able to upload a CSV to test it). You can skip these two. I'll review the article to see if it needs to be adjusted and come back to request those two images later if needed. Thank you!

**Comment by dubielzyk-expensify** [comment]
Also for the Company Card Settings, I don't quite follow this?
> For a workspace that has a pooled GL account, go to Settings > Workspaces > Group > [Workspace Name] > Connections > Expand the accounting integration, then add the orange highlight next to the Credit Card / Charge Card / Bank Transaction selector for the Non-reimbursable export option.

What does pooled GL mean?

**Comment by ren-jones** [comment]
@dubielzyk-expensify I think it just means any workspace that has an accounting integration set up. Then once you expand the integration on the Connections page, the selector will hopefully be present

**Comment by dubielzyk-expensify** [comment]
Where do I see this:
>then add the orange highlight next to the Credit Card / Charge Card / Bank Transaction selector for the Non-reimbursable export option.

**Comment by dubielzyk-expensify** [comment]
Nevermind. Found it. Needed a QBO integration :)

**Comment by dubielzyk-expensify** [comment]
I see this on QBO? Is that what you mean?
![CleanShot 2024-09-20 at 11 52 08@2x](https://github.com/user-attachments/assets/d8d196fd-f940-4b98-a395-a43b4981e1bf)

**Comment by ren-jones** [comment]
@dubielzyk-expensify Yes! That's the one! Thank you! Sorry, I didn't realize this was specific to QBO either 😬

**Comment by dubielzyk-expensify** [comment]
All good. Thanks for the quick answers. Here's the screens. Please double check and let me know if you need any changes.

[Here's the zip file](https://github.com/user-attachments/files/17069317/help-screens.zip)

![cardfeeds-01](https://github.com/user-attachments/assets/cdac19f5-734f-41f5-a712-d9770d41d49e)
![cardfeeds-02](https://github.com/user-attachments/assets/38972e45-f27c-49f8-9df9-ff783ef07cfd)
![compcard-01](https://github.com/user-attachments/assets/cba6901a-f4a1-4b96-9587-ae5faa07b6ce)
![compcard-02](https://github.com/user-attachments/assets/e8217632-dc2e-443f-9627-68811a2ce86d)
![compcard-03](https://github.com/user-attachments/assets/04cab4c3-cba1-495c-87ac-2a7b46e010cf)
![csv-01](https://github.com/user-attachments/assets/a046fd79-6e6f-417c-b9f3-559d7cb917ea)
![csv-02](https://github.com/user-attachments/assets/45c43348-2c2e-4395-aad2-517f9bc2b575)
![csv-03](https://github.com/user-attachments/assets/a8cd7ed2-a7c5-41d2-89f9-e4669bc57038)
![expenses-01](https://github.com/user-attachments/assets/57d11524-0b8a-43f6-85c3-e0d09e9a6b98)
![expenses-02](https://github.com/user-attachments/assets/5a201519-08c5-4afa-9963-ac6a1bee2be8)
![expenses-03](https://github.com/user-attachments/assets/dcac8c5a-62d4-4f29-b8c4-2d32cb2546fa)
![expenses-04](https://github.com/user-attachments/assets/76b6dab5-e381-42dd-905f-a2d2cb317d8f)
![expenses-05](https://github.com/user-attachments/assets/71830eba-43e2-4b7d-a36f-f584ff3e5904)
![tax_tracking-01](https://github.com/user-attachments/assets/6a198ed4-98e2-4023-ab02-f05f3776a6e3)
![tax_tracking-02](https://github.com/user-attachments/assets/46e180f5-8264-4b5e-84ab-45a1c440802b)


cc @Expensify/design for viz

**Comment by shawnborton** [comment]
Looks good to me! We might consider using a wider screen size (which I know would be different from the other standard sizes we've used) so that the card table looks a little nicer, but I don't feel strongly there.

**Comment by ren-jones** [comment]
These look great thank you!
